modules/metrics_prometheus_v1/src/main/scala/com/evolutiongaming/skafka/metrics/KafkaMetricsCollector.scala

```diff
@@ -71,12 +70,10 @@ class KafkaMetricsCollector[F[_]: Monad: ToTry](
     } yield new MetricSnapshots(result)
   }.toTry.get
 
-  protected def getPrometheusName(name: String, description: String, group: String): Option[String] = {
```

Comment:
The description and group were mixed up. I added named arguments to prevent this.